tests/atomistics/structure/test_atoms.py

```diff
@@ -1825,7 +1810,7 @@ def test_pymatgen_to_pyiron_conversion(self):
         pyiron_atoms_sd = pymatgen_to_pyiron(struct_with_sd)
 
         sd_equivalent = struct_with_sd.site_properties["selective_dynamics"] == [
-            x.selective_dynamics for x in pyiron_atoms_sd
+            x.selective_dynamics.tolist() for x in pyiron_atoms_sd
```

Comment:
I haven't run these tests myself, but just looking at the tests, is there a specific reason why some tests need to call `.tolist()` when they didn't before and why some have had .list() removed? I'm assuming that there is type weirdness happening with this change?

Reply:
I guess that's closely related to @pmrv's comment on list vs numpy.